src/app/components/ResumeItem.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { Box, Text, Flex } from "@chakra-ui/react";
+
+interface resumeItem {
+  title: string;
+  position?: string;
+  years?: string;
+  description?: string;
+  status?: string;
+}
+
```

Comment:
this should be named ResumeItemProps for clarity and in case you need to export them to be used elsewhere